### ISSUE TITLE
Flexihal PrintNC Without RGB

### ISSUE THREAD

**Issue by rcrules** [Expatria-Technologies/Flexi-HAL#9]
Would it be possible to make a PrintNC release without "STATUS_LIGHT_ENABLE=1", so all of the Aux pins are available for use?  Something like FLEXI_HAL_PRINTNC_NORGB_20240604_060724.uf2 or similar?  Thank You!!

**Comment by andrewmarles** [comment]
This build is what you describe:
https://github.com/Expatria-Technologies/STM32F4xx/releases/download/flexi-hal-v0.9.9.8/FLEXI_HAL_YGANG_ETH_SD_PRB_20231229_180124.1.uf2

**Comment by rcrules** [comment]
Thanks for the reply Andrew!!!

I have seen this request mentioned several times on the Discord, as of late.  I did try "FLEXI_HAL_YGANG_ETH_SD_PRB " after I saw it mentioned on the Discord, but I got some errors when restoring a setting.txt file, in IOSENDER, about incompatible features.

Also, according to the Build Configurations the "FLEXI_HAL_YGANG_ETH_SD_PRB" file doesn't contain SAFETY_DOOR_ENABLE=1.

Thanks again for the consideration!!
